hackathon_site/dashboard/frontend/src/components/general/ProductOverview/ProductOverview.js

```diff
@@ -0,0 +1,228 @@
+    };
+
+    return (
+        handleSubmit && (
+            <Formik
+                initialValues={{ quantity: "1" }}
+                onSubmit={onSubmit}
+                validateOnBlur={false}
+                validationOnChange={false}
+                validationSchema={false}
+            >
+                {(formikProps) => (
+                    <CartForm
+                        availableQuantity={availableQuantity}
+                        handleSubmit={formikProps.handleSubmit}
+                        handleChange={formikProps.handleChange}
+                        requestFailure={requestFailure}
+                        values={formikProps.values}
+                    />
+                )}
+            </Formik>
+        )
+    );
+};
+
+const DetailInfoSection = ({
+    manufacturer,
+    model_num,
+    datasheet,
+    notes,
+    constraints,
+}) => {
+    return (
+        <div className={styles.detailinfosection}>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2" className={styles.heading}>
+                    Constraints
+                </Typography>
+                {constraints.map((constraint, i) => (
+                    <Typography key={i} variant="body2">
+                        {constraint}
+                    </Typography>
+                ))}
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2" className={styles.heading}>
+                    Manufacturer
+                </Typography>
+                <Typography>{manufacturer}</Typography>
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2" className={styles.heading}>
+                    Model Number
+                </Typography>
+                <Typography>{model_num}</Typography>
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2" className={styles.heading}>
+                    Datasheet
+                </Typography>
+                <Link
+                    href={datasheet}
+                    rel="noopener"
+                    color="inherit"
+                    underline="none"
+                    target="_blank"
+                    className={styles.bodyinfoDataSheet}
+                >
+                    <LaunchIcon></LaunchIcon>
+                    <Typography>Link</Typography>
+                </Link>
+            </div>
+            <div className={styles.bodyinfo}>
+                <Typography variant="body2" className={styles.heading}>
+                    Notes
+                </Typography>
+                {notes.map((note, i) => (
+                    <Typography key={i} variant="body2">
+                        {note}
+                    </Typography>
+                ))}
+            </div>
+        </div>
+    );
+};
+
+const MainSection = ({ type, name, total, available, categories, img }) => {
```

Comment:
would rename the `available` prop to `quantityAvailable`, just so it's not confused as a bool